ouroboros-consensus/test/storage-test/Test/Ouroboros/Storage/ChainDB/StateMachine.hs

```diff
@@ -912,9 +912,9 @@ generator ::
 generator loe genBlock m@Model{..} =
   At
     <$> frequency
-      [ (30, genAddBlock)
+      [ (100, genAddBlock)
```

Comment:
We might want to add here a comment similar to what's mentioned on `983856001a0f87dae23fef584270b9b5b4b395b6` about the justification for this frequency change.